elevation_topic: '/elevation_mapping/elevation_map_raw'
height_layer: 'elevation'
target_frame_id: 'odom'
submap:
  width: 8.0
  length: 8.0
publish_to_controller: true
frequency: 1.0
